ldap/servers/plugins/replication/replutil.c

```diff
@@ -382,6 +384,9 @@ parse_changes_string(char *str)
             }
             line = ldif_getline(&next);
         }
+        if (!mod_consumed) {
+            slapi_mod_done(&mod);
```

Comment:
IMHO the mod should also be added to smod if it is valid ...
a minor point,  I wonder if it would not be easier to remove the  mod_consumed variable |
and simply reset the mod with slapi_mod_init(&mod, 0); after consuming it.
And do a systematic slapi_mod_done here ... (still need to add the last mod if it is valid ...)

Reply:
Let me look into this...

Reply:
Fixed.  Covscan and ASAN do not complain with the latest change